test/runTests.cpp

```diff
@@ -79,7 +79,7 @@ int main(int argc, char* argv[]) {
 
 void fillRandomly(char* seq, int seqLength, int alphabetLength) {
     for (int i = 0; i < seqLength; i++)
-        seq[i] = (char) rand() % alphabetLength;
+        seq[i] = static_cast<char>(rand()) % alphabetLength;
```

Comment:
Hm I am not sure if this one should be applied just to rand() or to whole expression (rand % alphabeLength)? How was it working before, does C style cast have precedence over %?

Reply:
When I build this with GCC 9 and `-Werror=old-style-cast` I get the following diagnostic (which I followed when correcting):

```
/Users/Soap/Desktop/edlib/test/runTests.cpp: In function 'void fillRandomly(char*, int, int)':
/Users/Soap/Desktop/edlib/test/runTests.cpp:82:30: error: use of old-style cast to 'char' [-Werror=old-style-cast]
   82 |         seq[i] = (char) rand() % alphabetLength;
      |                              ^
      |                  -------------
      |                  static_cast<char> ( rand())
```

If you check the [C++ operator precedence rules](https://en.cppreference.com/w/cpp/language/operator_precedence), you will see that the C-style cast takes precedence over the modulo operator.